### DISCUSSION TITLE
[FEATURE] Document How Frequently Reports can be requested

### DISCUSSION THREAD

**Discussion by kpconnell** [amzn/selling-partner-api-models#601] [Ideas]
**Describe the improvement**
The current behavior of the reports api is if you request a report "too frequently" you will get a fatal status back. For example, GET_FBA_FULFILLMENT_REMOVAL_SHIPMENT_DETAIL_DATA  appears to only be requestable approximately once every two hours.  When building historical data, it is a significant challenge to deal with reports that behave like this particularly when the status is non-intuitive (see #1329 ) and the frequency that results in cancelled or fatal is not documented.    The improvement would be to include in reporttypes enum documentation the limitations around how often you can request a report. 

**URL of the content on GitHub**
https://developer-docs.amazon.com/sp-api/docs/report-type-values

**Description of what the documentation should say**
See above

**Comment by kpconnell** [comment]
Report Ids (amazon.com marketplace) that exhibit this behavior:

** Report 442999019351 (GET_FBA_FULFILLMENT_REMOVAL_SHIPMENT_DETAIL_DATA) never finished Array ( [request_json] => {"report_options":null,"report_type":"GET_FBA_FULFILLMENT_REMOVAL_SHIPMENT_DETAIL_DATA","data_start_time":"2022-07-24","data_end_time":"2022-08-23","marketplace_ids":["ATVPDKIKX0DER"]} [storeid] => xxxxx [reportId] => 442999019351 [marketplaceId] => ATVPDKIKX0DER [reportType] => GET_FBA_FULFILLMENT_REMOVAL_SHIPMENT_DETAIL_DATA [sellerId] => A2XHA54xxxxx [status] => FATAL [reportDocumentId] => )

Requesting Report xxxxx GET_FBA_FULFILLMENT_REMOVAL_SHIPMENT_DETAIL_DATA 2022-08-23 - 2022-09-22

** Report 443000019351 (GET_FBA_FULFILLMENT_REMOVAL_SHIPMENT_DETAIL_DATA) never finished Array ( [request_json] => {"report_options":null,"report_type":"GET_FBA_FULFILLMENT_REMOVAL_SHIPMENT_DETAIL_DATA","data_start_time":"2022-08-23","data_end_time":"2022-09-22","marketplace_ids":["ATVPDKIKX0DER"]} [storeid] => xxxxx [reportId] => 443000019351 [marketplaceId] => ATVPDKIKX0DER [reportType] => GET_FBA_FULFILLMENT_REMOVAL_SHIPMENT_DETAIL_DATA [sellerId] => A2XHA54xxxxx [status] => FATAL [reportDocumentId] => )

Requesting Report xxxxx GET_FBA_FULFILLMENT_REMOVAL_SHIPMENT_DETAIL_DATA 2022-09-22 - 2022-10-21

** Report 443001019351 (GET_FBA_FULFILLMENT_REMOVAL_SHIPMENT_DETAIL_DATA) never finished Array ( [request_json] => {"report_options":null,"report_type":"GET_FBA_FULFILLMENT_REMOVAL_SHIPMENT_DETAIL_DATA","data_start_time":"2022-09-22","data_end_time":"2022-10-21","marketplace_ids":["ATVPDKIKX0DER"]} [storeid] => xxxxx [reportId] => 443001019351 [marketplaceId] => ATVPDKIKX0DER [reportType] => GET_FBA_FULFILLMENT_REMOVAL_SHIPMENT_DETAIL_DATA [sellerId] => A2XHA54xxxxx [status] => FATAL [reportDocumentId] => )

Requesting Report xxxxx GET_FBA_FULFILLMENT_REMOVAL_SHIPMENT_DETAIL_DATA 2022-10-21 - 2022-11-20

** Report 443002019351 (GET_FBA_FULFILLMENT_REMOVAL_SHIPMENT_DETAIL_DATA) never finished Array ( [request_json] => {"report_options":null,"report_type":"GET_FBA_FULFILLMENT_REMOVAL_SHIPMENT_DETAIL_DATA","data_start_time":"2022-10-21","data_end_time":"2022-11-20","marketplace_ids":["ATVPDKIKX0DER"]} [storeid] => xxxxx [reportId] => 443002019351 [marketplaceId] => ATVPDKIKX0DER [reportType] => GET_FBA_FULFILLMENT_REMOVAL_SHIPMENT_DETAIL_DATA [sellerId] => A2XHA54xxxxx [status] => FATAL [reportDocumentId] => )

Requesting Report xxxxx GET_FBA_FULFILLMENT_REMOVAL_SHIPMENT_DETAIL_DATA 2022-11-20 - 2022-12-19

** Report 443003019351 (GET_FBA_FULFILLMENT_REMOVAL_SHIPMENT_DETAIL_DATA) never finished Array ( [request_json] => {"report_options":null,"report_type":"GET_FBA_FULFILLMENT_REMOVAL_SHIPMENT_DETAIL_DATA","data_start_time":"2022-11-20","data_end_time":"2022-12-19","marketplace_ids":["ATVPDKIKX0DER"]} [storeid] => xxxxx [reportId] => 443003019351 [marketplaceId] => ATVPDKIKX0DER [reportType] => GET_FBA_FULFILLMENT_REMOVAL_SHIPMENT_DETAIL_DATA [sellerId] => A2XHA54xxxxx [status] => FATAL [reportDocumentId] => )

**Comment by teddy-codes** [comment]
We are currently running into the same problem. I hope that this can be fixed before our second required deprecation date which is December 31st (basically just one day...). How often are reports available to run?

@chapmanjw Do you know anyone who is responsible for this? This is a p0 issue for us. Cheers!
